test/validation/api/packet/packet.c

```diff
@@ -827,6 +827,16 @@ static void packet_test_context(void)
 	CU_ASSERT(odp_packet_user_ptr(pkt) == NULL);
 }
 
+static void packet_test_payload_offset(void)
+{
+	odp_packet_t pkt = test_packet;
+
+	CU_ASSERT(odp_packet_payload_offset_set(pkt, 42) == 0);
```

Comment:
The min and max values (0 ... odp_packet_len()-1) could also be tested.

Reply:
fixed